packages/fxa-auth-server/lib/error.js

```diff
@@ -599,7 +599,7 @@ AppError.smsSendRateLimitExceeded = () => {
     code: 429,
     error: 'Too many requests',
     errno: ERRNO.SMS_SEND_RATE_LIMIT_EXCEEDED,
-    message: 'Client has sent too many requests',
+    message: 'Text message limit reached',
```

Comment:
Were these error message updates approved by content or UX? Just double checking. I think the `backendServiceFailure` message updates apply to multiple areas including SubPlat.

Reply:
UX/Content haven't holistically reviewed our general error messages.

FWIW though, `System unavailable, try again soon` sounds more user-friendly to me.

We also have the option of showing a different or more specific error message on the frontend if we want. E.g., for recovery method choice page I'm checking for the error code and replacing these generic messages with the following to more closely align with Figma designs

![image](https://github.com/user-attachments/assets/b30253cf-bce4-4f74-9982-b78dda090757)

Reply:
@LZoog btw, the `System unavailable, try again soon` was actually used for the `ERRNO.BACKEND_SERVICE_FAILURE` on line 300. These changes just make it consistent.